src/components/Header/HeaderNotification/HeaderNotification.js

```diff
@@ -39,7 +40,10 @@ const HeaderNotification = ({
     </a>
     <div className={`${namespace}__wrapper`}>
       <button
-        className={`${namespace}__button`}
+        className={classnames(
+          `${namespace}__button`,
+          `${namespace}__button--clear`
```

Comment:
I'm curious to hear more about the rationale behind applying a new class here, versus adding the styles to `${namespace}__button`

Reply:
There are other buttons that share the same class but don’t need these particular styles. The base class provides a number of general styles for all header buttons, and this extra class provides some specific overrides.

Reply:
As far as I recall, this class will output as `security--header__notification__button`, which should be scoped to this component